### ISSUE TITLE
billing/billing-entries

### ISSUE THREAD

**Issue by jerzakm** [allegro/allegro-api#8835]
Czy jest możliwość pobrania przez API szczegółów dotyczących danego BillingEntry. Chodzi mi tutaj o dane z kolumny "Szczegóły operacji" która jest w pliku csv z Historią operacji które każdy może wygenerować w moje-allegro/moje-konto/rachunki/rozliczenia-z-allegro

Zależy nam głównie na danych o przesyłkach kurierskich. Nr listów przewozowych itp. Przykładowo wygląda to tak:

"Numer nadania: asdasdas, Kod: ONE_BOX_P5, Identyfikator zamówienia: uuid, Szczegóły: Allegro One Box - dostawa dziś; Order value = 99.99PLN - w ramach Uslugi Allegro Smart"

**Comment by AureliuszBrussy** [comment]
Nie, nie udostępniamy takiej możliwości. Mamy podobną sugestię z [zakładce Discussions](https://github.com/allegro/allegro-api/discussions/7666).  Możesz dołączyć swój głos w dyskusji.
